amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2558,6 +2558,7 @@ VX_API_ENTRY vx_node VX_API_CALL vxExtRppPreemphasisFilter(vx_graph graph, vx_te
     return node;
 }
 
+
```

Comment:
Please remove the additional blank line added

Reply:
done